integration-tests/discovery-nacos-tests/order-consumer/src/main/resources/bootstrap.yaml

```diff
@@ -31,6 +31,7 @@ spring:
         metadata:
           x-test: value
           x-test2: value2
+          serviceName: order
```

Comment:
do not configure this. service name should use actual service name not property

Reply:
fixed